bin/panic-button.sh

```diff
@@ -1,13 +1,15 @@
 #!/usr/bin/env bash
 
-# Trashes all build folders and `node_modules`
+# Trashes all `node_modules` folders, lockfiles and build artifacts to give you
+# a clean version of the project.
 
-printf 'Trashing `node_modules` folders and build artifacts...\n'
+printf 'Trashing `node_modules` folders, lockfiles and build artifacts...\n'
 
 ROOT_DIR="$(cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null && cd .. && pwd)"
 
 printf "Cleaning \`$ROOT_DIR\`..."
 rm -rf $ROOT_DIR/node_modules
+rm -rf $ROOT_DIR/yarn.lock
```

Comment:
Why is the lock file being deleted?

Reply:
Force of habit. In npm, it gets rewritten all the time anyway; I think yarn is a bit more conservative. I'll leave it in.